system-tests/projects/no-specs-custom-pattern/cypress.config.js

```diff
@@ -11,6 +11,6 @@ module.exports = {
   },
   e2e: {
     supportFile: false,
-    specPattern: 'src/**/*.cy.{js,jsx}',
+    specPattern: 'src/**/*.{cy,spec}.{js,jsx}',
```

Comment:
Did you intent to check this in?

Reply:
Yup, so, the new test makes sense - waiting the CI to check if it breaks something - but the current spec file passed